kratos/python/add_properties_to_python.cpp

```diff
@@ -177,6 +227,16 @@ void  AddPropertiesToPython(pybind11::module& m)
     .def("HasVariables", &Properties::HasVariables)
     .def("HasTables", &Properties::HasTables)
     .def("IsEmpty", &Properties::IsEmpty)
+    .def("NumberOfSubproperties", &Properties::NumberOfSubproperties)
+    .def("AddSubProperty", &Properties::AddSubProperty)
```

Comment:
```
    .def("AddSubProperties", &Properties::AddSubProperties)
```
would be more consistent => the class is called `Properties`
I understand thought why you did is like this, so I am not sure

Reply:
Indeed "typo"